source/frontends/libretro/diskcontrol.cpp

```diff
@@ -28,13 +28,23 @@ namespace
     return std::equal(prefix.begin(), prefix.end(), value.begin());
   }
 
+  static std::string fileStem(const std::string& path)
```

Comment:
I'm not sure if the msys2 compiler doesn't support c++17, or if the CMake setting isn't being honored, but the compiler couldn't find `string::stem` or `string::native`.